### ISSUE TITLE
[BUG] 

### ISSUE THREAD

**Issue by camargojib** [rubin-dp0/Support#57]
**Describe the bug**
After logging in at the Rubin Science Platform Portal, 

(this address: https://data.lsst.cloud/   plus a click on "Portal")

using my ORCID as ID, I get the
an error message (please, see attached file). It makes no difference to select or not DP0.3
<img width="1274" alt="Screen Shot 2023-09-30 at 18 38 16" src="https://github.com/rubin-dp0/Support/assets/146592805/d6c26e23-2f15-461f-846b-7718f43806e9">

**To Reproduce**
Every time I log in at the Rubin Science Platform Portal.

**Desktop (please complete the following information):**
 MacOS X El Capitan (10.11.6)

Thanks for your help!
-Julio

<img width="1274" alt="Screen Shot 2023-09-30 at 18 38 16" src="https://github.com/rubin-dp0/Support/assets/146592805/af0abd76-99b8-4ad3-8442-e84a40fb38cc">

**Comment by MelissaGraham** [comment]
Hi Julio,
I'm assigning @frossie to this issue because I cannot recreate it and from what I can see of your user account (Julio Ignacio Bueno de Camargo) it all looks normal. Could you add which browser you're using and whether you were ever able to log into the Portal Aspect? If this is a newly occurring problem, did it start the day you submitted this issue or has it happened before?
Thanks very much,
Melissa

**Comment by camargojib** [comment]
Dear Melissa,

           Thanks for your reply.

> Could you add which browser you're using and whether you were ever able
to log into the Portal Aspect?
Browser: Firefox 78.15.0esr (64-bit)
I logged into the portal during the DP0.3 Virtual Kick-Off Session.

> If this is a newly occurring problem, did it start the day you submitted
this issue or has it happened before?
It happened a few days before I took the problem to you. It has never
happened before, but the before was
that of the first time ( Virtual Kick-Off Session).

Best regards,
-Julio


Em seg., 2 de out. de 2023 às 12:09, Melissa Graham <
***@***.***> escreveu:
